Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.csproj

```diff
@@ -8,7 +8,7 @@
 	  <AssemblyName>Frends.SFTP.DownloadFiles</AssemblyName>
 	  <RootNamespace>Frends.SFTP.DownloadFiles</RootNamespace>
 
-	  <Version>2.10.0</Version>
+	  <Version>2.10.1</Version>
```

Comment:
I may be mistaking, but this probably should be 2.11.0. Third number is for testing and developing usually.